### PR TITLE
Add GitHub Actions CI

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      - name: Lint
+        run: |
+          pip install pre-commit
+          if pre-commit run --from-ref origin/master --to-ref HEAD; then
+              echo "Linting errored; this will be fatal sometime in the near future"
+          fi
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
```
Comment (profjsb) [inline .github/workflows/test.yml:30]: We should also be checking 3.7 (I believe we require >=3.7).

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+on: [push]
+
+jobs:
+  test:
+    name: Test SkyPortal
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11.8
```
Comment (profjsb) [inline .github/workflows/test.yml:12]: I thought we're on postgres 12.x -- please confirm.

File: package.json [hunk 3]
```diff
@@ -11,21 +11,24 @@
...
-    "material-ui-phone-number": "2.2.6",
     "bokehjs": "0.12.9",
     "check-dependencies": "1.1.0",
     "clsx": "1.1.1",
+    "core-js": "3.6.5",
     "css-loader": "4.2.0",
     "dayjs": "1.8.31",
     "exports-loader": "0.7.0",
     "imports-loader": "0.8.0",
+    "material-ui-phone-number": "2.2.6",
```
Comment (acrellin) [inline package.json:22]: Should these changes be in here?

File: .github/workflows/test.yml (hunk 1 above)
Comment (stefanv) [reply .github/workflows/test.yml:36]: Switched to 3.7 to test our minimal version of Python.

File: .github/workflows/test.yml (hunk 2 above)
Comment (stefanv) [reply .github/workflows/test.yml]: I'm now using postgres latest

File: package.json (hunk 3 above)
Comment (stefanv) [reply package.json:22]: Yes, without these extra side dependencies, npm installs fail.

**Comment by stefanv** [comment]
I added the relevant CI keys.

File: doc/dev.md [hunk 4]
```diff
@@ -79,3 +79,14 @@ We use ESLint to ensure that our JavaScript & JSX code is consistent and conform
 ## Docker images
 
 Run `make docker-images` to build and push to Docker hub.
+
+## Debugging GitHub Actions
+
+You can insert the following step to debug your workflow:
+
+```yaml
```
Comment (dannygoldstein) [inline doc/dev.md:87]: Love this

**Pull request merged**